packages/lib-classifier/src/plugins/drawingTools/models/marks/TemporalPoint/TemporalPoint.js

```diff
@@ -0,0 +1,43 @@
+import {
+  addDisposer,
+  getRoot,
+  getParentOfType,
+  isValidReference,
+  types
+} from 'mobx-state-tree'
+import { TemporalPoint as TemporalPointComponent } from '../../../components'
+import { TemporalPointTool } from '@plugins/drawingTools/models/tools'
+
+import Mark from '../Mark'
+import Point from '../Point'
+
+const TemporalPointModel = types
+  .model('TemporalPointModel', {
+    t: types.optional(types.number, 0)
```

Comment:
I prefer object properties and variable names to be spelled out, so it's absolutely clear as possible on intent. How about `time` here?

Reply:
`time` is good for this iteration. 👍

Though I suspect in the future this will evolve into `startTime` and `endTime` in a later enhancement. (Time _ranges_ appears to be the direction the research project's requirements are going, IIRC?)